interface/rak13006/RAK13006-MCP2518/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 RAKwireless

```

Comment:
Should be 2022

Reply:
@xoseperez yes, the [RAK13006-MCP2518](https://github.com/RAKWireless/RAK13006-MCP2518) library is published on the RAKWireless repo, so I need to remove this library and tell the user they can download the latest library from RAKWireless repo in the README.md file，right?

Reply:
Right